dev-manuals/services/overworld/database-model-and-type.md

```diff
@@ -48,8 +49,10 @@
 
 ## Player-specific data
 - the player-specific data should also be stored in a SQL database, there are relations from unlocked areas to the defined areas, completed dungeons, tasks, ...
-- in the future, if database updates happen very often we can switch to some NoSQL databases for faster queries.
 
 # Result
 
-SQL is the favorite here because of the many relations we have. For our project in the overworld, it does not make much difference whether we use MySQL or PostgreSQL, because the performance does not have many differences and the features of both databases are completely sufficient for us
\ No newline at end of file
+SQL is the favorite here because of the many relations we have. For our project in the overworld, it does not make much difference whether we use MySQL or PostgreSQL, because the performance does not have many differences and the features of both databases are completely sufficient for us
```

Comment:
```suggestion
SQL is the favorite here because of the many relations we have. For our project, it does not make a difference what type of database system we use because the performance will be similar and the features of all databases are sufficient for us.
```